### DISCUSSION TITLE
这个版本 安卓能正常关注 注册吗？

### DISCUSSION THREAD

**Discussion by chongqing0202** [Semporia/TikTok-Unlock#257] [Q&A]
我下载的官方版 无法关注 账号密码登录有时也不行～

**Comment by seoho8908** [comment]
> 我下载的官方版 无法关注 账号密码登录有时也不行～

安卓手机使用TikTok必须破解版。要不就是拔SIM卡，或者插外国手机卡。

**Reply by Ledrong** [reply]
你好，我是來自俄羅斯的學生。我想在https://buff.163.com/平台進行交易。但為此我需要驗證我的帳戶，而這只能使用支付寶和steam來完成。請您只需要驗證一下，一切都是安全的。我會付1000元
